go.mod

```diff
@@ -55,7 +55,7 @@ require (
 	github.com/go-mysql-org/go-mysql v1.5.0 // replaced
 	github.com/go-piv/piv-go v1.11.0
 	github.com/go-redis/redis/v9 v9.0.0-rc.1 // replaced
-	github.com/go-resty/resty/v2 v2.3.0
+	github.com/go-resty/resty/v2 v2.7.0
```

Comment:
This is a big jump, I wonder what happened here.

I don't see anything particularly in the release notes, but I'd be OK pushing this one to a separate PR too.